src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/ProjectComponents.cs

```diff
@@ -15,5 +15,7 @@ public enum ProjectComponents
         Wpf = 1 << 3,
         WinForms = 1 << 4,
         AspNetCore = 1 << 5,
+        XamarinAndroid = 1 << 6,
+        XamariniOS = 1 << 7,
```

Comment:
Do we want to add a Maui component as well?

Reply:
I figured I'd add them incrementally as I add the next steps. next PR will include the MAUI components + checks for them so that it adds the correct templates. I guess I could add them now but it won't be used anywhere at the moment (relative to this PR at least).

Reply:
@twsouthwick wanted to check if the reason makes sense to you? made all the other suggested updates! 😄

Reply:
That's fine with me!